ntbs-service/DataAccess/TestResultRepository.cs

```diff
@@ -0,0 +1,21 @@
+using System.Linq;

```

Comment:
This looks like a completely new class rather than a refactor as the old testresultrepository was in the wrong directory (services instead of dataaccess)